Assets/Tests/InputSystem/CoreTests_Devices.cs

```diff
@@ -2849,15 +2849,6 @@ public void Devices_CanGetNameOfCurrentKeyboardLayout()
         Assert.That(keyboard.keyboardLayout, Is.EqualTo("new"));
     }
 
-    [Test]
-    [Category("Devices")]
-    public void Devices_CanGetKeyCodeFromKeyboardKey()
```

Comment:
This became obsolete since it was replaced bye a more elaborate test `Devices_Keyboard_CanGetKeyCodeFromKeyboardKey` that verifies this for all keys and not only `A`.